22-web-infrastructures/COURSE_MATERIAL.md

```diff
@@ -65,90 +64,194 @@ system.
 Thanks to HTTP features, the reverse proxy will be able to serve multiple
 domains on the same IP address.
 
-You will also learn how to use caching to improve the performance of your
-system.
+Finally, you will learn how to monitor your system.
 
-Finally, you will learn how to monitor your system and how to calculate the
-number of servers needed to handle a certain amount of requests.
+As this chapter is quite abstract, you will first setup your environment to be
+able to run some code examples along with the theory.
 
 ## Prepare and setup your environment
 
-### Access your `hosts` file
-
-In this section, you will access your `hosts` file.
-
-The host file is a computer file used by an operating system to map hostnames to
-IP addresses. The hosts file is a plain text file and is conventionally named
-`hosts`. The hosts file can be used as an alternative to (or in conjunction
-with) DNS.
+### Clone the code examples repository on the virtual machine
 
-On Unix-like operating systems (Linux and macOS), the hosts file is located at
-`/etc/hosts`.
+In this section, you will clone the
+[`heig-vd-dai-course/heig-vd-dai-course-code-examples`](https://github.com/heig-vd-dai-course/heig-vd-dai-course-code-examples)
+repository on the virtual machine.
```

Comment:
I would specify that it refers to the VM created in the previous chapter on Azure

Reply:
True, thank you!